workflow/Snakefile

```diff
@@ -1,6 +1,8 @@
 import os
 import os.path as osp
 
+configfile: "config/snakemake/roman.yaml"
```

Comment:
Since we have multiple snakemake configs now I just run my stuff with `snakemake --configfile config/snakemake/roman.yaml ...` instead of defining it in Snakefile

Reply:
Thank you